.github/workflows/rust.yml

```diff
@@ -49,6 +44,10 @@ jobs:
       env:
         LLVM_PROFILE_FILE: coverage/all-features-%p-%m.profraw
       run: cargo test --all-features
+    - name: Run tests (all features) + benchmarks
```

Comment:
Only runs the benchmarks with 1 iteration, as though they are tests.  So it checks that they work without taking 5 minutes.